src/sql/parts/notebook/notebookActions.ts

```diff
@@ -38,6 +40,51 @@ export class AddCellAction extends Action {
 	}
 }
 
+export class TrustedAction extends Action {
+	//Constants
```

Comment:
nitpick: as mentioned above the coding style is to have a space after comment `// Constants`. Also fix elsewhere in the file